delete_projects.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require 'functions.php';
+
+$db = createDB();
+
+$stmt = $db->prepare(
+    "DELETE FROM `projects`
+      WHERE `id` = :id;"
+);
+$stmt->bindParam(':id',    $_GET["id"]);
+$stmt->execute();
+
+echo 'Project successfully deleted';
```

Comment:
why not echo inside the html body and make your file valid?

Reply:
Done.